### PR TITLE
PMM-12961 [Tech Preview] New Databases Overview Dashboard

### PR DESCRIPTION
**Pull request by nethalo**
[PMM-12961](https://perconadev.atlassian.net/browse/PMM-12961)
New experimental GLOBAL overview dash

[PMM-12961]: https://perconadev.atlassian.net/browse/PMM-12961?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

image

### CODE REVIEW

**Review by rnovikovP — APPROVED** [review]

File: dashboards/Experimental/databases_overview.json [hunk 1]
```diff
@@ -0,0 +1,1055 @@
...
+          "type": "dashboard"
+        },
+        "type": "dashboard"
+      }
+    ]
+  },
+  "editable": true,
+  "fiscalYearStartMonth": 0,
+  "graphTooltip": 1,
+  "id": 78,
```
Comment (BupycHuk) [inline dashboards/Experimental/databases_overview.json:27]: can we drop `id` fields here? Here and in subpanels

**Review by BupycHuk — APPROVED** [review]

**Comment by BupycHuk** [comment]
@rnovikovP what do you think should it be called `QAN overview` or something else, as far as I see it uses only QAN data and don't use exporters metrics.

File: dashboards/Experimental/databases_overview.json (hunk 1 above)
Comment (ademidoff) [inline dashboards/Experimental/databases_overview.json:27]:
```suggestion
  "id": null,
```

Comment (ademidoff) [reply dashboards/Experimental/databases_overview.json:27]: @nethalo You can just replace the integers with `null`, so in this case it'd become `"id": null,`

**Comment by rnovikovP** [comment]
> @rnovikovP what do you think should it be called `QAN overview` or something else, as far as I see it uses only QAN data and don't use exporters metrics.

It's not only QAN. CH datasource here because Metrics is default and removed.  
The final name - will be set when we do a GA after feedback.

**Pull request merged**